datafusion/proto/proto/datafusion.proto

```diff
@@ -526,7 +526,7 @@ message WindowExprNode {
     string udaf = 3;
     string udwf = 9;
   }
-  LogicalExprNode expr = 4;
+  repeated LogicalExprNode exprs = 4;
```

Comment:
**<ins>This change is not backwards compatible!</ins>**

This is necessary to correctly serialize expression functions like `lag(col("a"), Some(2), Some(100))` which has more than 1 argument. 

Without this gets serialized into `lag(col("a"))` ignoring the second, and third arguments and results in an error when deserialized.

Reply:
Happy to redo this if not the right way of handling this for downstream DataFusion users.

Reply:
I think it is fine as the protobuf isn't guaranteed to be compatible across versions, see https://docs.rs/datafusion-proto/latest/datafusion_proto/#version-compatibility